templates/sidebar-news.twig

```diff
@@ -2,13 +2,25 @@
       <aside class="aside-right col-sm-3">
         {{sidebar_top}}
         {% if post.baixades_relacionades_noticia %}
-          <div class="caixa-gris">
           {% if post.baixades_relacionades_noticia is iterable %}
-              {% for baixada in post.baixades_relacionades_noticia %}
-              <a href="{{ baixada }}"><i class="fa fa-download"></i><span>{{ baixada|get_caption_from_media_url }}</span></a>
+          {% for baixada in post.baixades_relacionades_noticia %}
```

Comment:
Aquest `for` té molta lògica... No hi ha manera de que la informació vinga de `baixades_relacionades_notícia`? 

La llista de baixades hauria d'estar filtrada, per a només incloure les que tinguen el caption definit, i amb això es simplificaria molt els `if`-`else`